src/SFA.DAS.EAS.Portal.Client/SFA.DAS.EAS.Portal.Client.csproj

```diff
@@ -14,6 +14,7 @@
         <PublishRepositoryUrl>true</PublishRepositoryUrl>
         <IncludeSymbols>true</IncludeSymbols>
         <SymbolPackageFormat>snupkg</SymbolPackageFormat>
+        <Version>1.6.491</Version>
```

Comment:
The version of the package is injected as part of the Azure Devops package release pipeline. This line should be removed.

Reply:
OK

Reply:
👍